examples/Microsoft.Spark.CSharp.Examples/Sql/MachineLearning/SentimentAnalysis.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.Spark.Sql;
+
+namespace Microsoft.Spark.Examples.Sql.MachineLearning
+{
+    /// <summary>
+    /// Example of using ML.NET + .NET for Apache Spark
+    /// for sentiment analysis.
+    /// </summary>
+    internal sealed class SentimentAnalysis : IExample
+    {
+        public void Run(string[] args)
+        {
+            // Change this flag once you've set up
+            // the ML.NET dependencies described in the README:
+            // Update url to location of MLModel.zip (line 90)
+            // Copy ML.NET dlls to your project's folder
+            int dependenciesDone = 0;
+
+            if (args.Length != 1)
+            {
+                Console.Error.WriteLine(
+                    "Usage: SentimentAnalysis <path to review dataset>");
+
+                Environment.Exit(1);
+            }
+
+            if (dependenciesDone == 0)
+            {
+                Console.Error.WriteLine(
+                    "You need to complete setting ML.NET dependencies");
+
+                Environment.Exit(1);
+            }
+
+            SparkSession spark = SparkSession
+                .Builder()
+                .AppName(".NET for Apache Spark Sentiment Analysis")
+                .GetOrCreate();
+
+            DataFrame df = spark
+                .Read()
+                .Option("header", true)
+                .Option("inferSchema", true)
+                .Csv(args[0]);
+            df.Show();
+
+            // Use ML.NET to evaluate each review 
+            spark.Udf().Register<string, bool>(
+                "MLudf",
+                (text) => Sentiment(text));
+
+            df.CreateOrReplaceTempView("Reviews");
+            DataFrame sqlDf = spark.Sql("SELECT Column1, MLudf(Column1) FROM Reviews");
+            sqlDf.Show();
+
+            // Print out first 20 rows of data
+            // Prevents data getting cut off (as it is when we print a DF)
+            IEnumerable<Row> rows = sqlDf.Collect();
```

Comment:
You can use the [`Limit`](https://github.com/dotnet/spark/blob/67451fa1179e9d9667d96c40fc9ae5b222dcea8d/src/csharp/Microsoft.Spark/Sql/DataFrame.cs#L483) method

Reply:
Used `Take(20)` instead (so that it would return an `IEnumerable<Row>`)